tox.ini

```diff
@@ -184,6 +184,31 @@ commands_pre=
 commands=
     yea {env:CI_PYTEST_SPLIT_ARGS:} --strict --shard {env:SHARD:default} run {posargs:--all}
 
+[testenv:reports-py{38,39,310,311,312}]
+deps=
+    -r{toxinidir}/requirements_test.txt
```

Comment:
do you need all the deps in requirement_test? i wonder if you can save some time